sdk-webview/src/main/java/com/reach5/identity/sdk/webview/WebViewProviderConfig.kt

```diff
@@ -11,18 +11,18 @@ import java.net.URLEncoder
 internal data class WebViewProviderConfig(
     val providerConfig: ProviderConfig,
     val sdkConfig: SdkConfig,
-    val origin: String
+    val origin: String,
+    val scope: String
 ) : Parcelable {
 
     fun buildUrl(pkce: Pkce): String {
-        val scope = (providerConfig.scope)
         val params = mapOf(
             "client_id" to sdkConfig.clientId,
             "provider" to providerConfig.provider,
             "origin" to origin,
             "redirect_uri" to sdkConfig.scheme,
             "response_type" to "code",
-            "scope" to scope.joinToString(" "),
+            "scope" to scope,
```

Comment:
I think the main culprit is here, both in `master` and in the current branch. This `scope` value refers to _**our**_ OAuth2 flow, and it should never be mixed with the scope value configured for the provider in the console. To override the Provider scope, the `provider_scope` value should be used instead.

If my understanding is correct, the main issue at stake here is obtaining a **Refresh Token**, meaning we need the **offline_access** scope.
- we need to be able to configure the scope value: OK
- but we shouldn't concatenate the provided scope value with the configured provider scope

Reply:
```suggestion
            "scope" to scope,
            "provider_scope" to providerConfig.scope
```